charts/stable/vikunja/values.yaml

```diff
@@ -17,48 +10,23 @@ workload:
           probes:
             liveness:
               type: http
-              port: 3456
+              port: 10220
               path: /health
             readiness:
               type: http
-              port: 3456
+              port: 10220
               path: /health
             startup:
               type: http
-              port: 3456
+              port: 10220
               path: /health
-        frontend:
-          enabled: true
-          imageSelector: frontendImage
-          probes:
-            liveness:
-              type: http
-              port: 80
-            readiness:
-              type: http
-              port: 80
-            startup:
-              type: http
-              port: 80
-        proxy:
-          enabled: true
-          imageSelector: nginxImage
-          probes:
-            liveness:
-              type: http
-              port: "{{ .Values.service.main.ports.main.port }}"
-            readiness:
-              type: http
-              port: "{{ .Values.service.main.ports.main.port }}"
-            startup:
-              type: http
-              port: "{{ .Values.service.main.ports.main.port }}"
 vikunja:
   service:
     jwtttl: 259200
     jwtttllong: 2592000
     motd: Welcome to your new Vikunja instance
     frontendurl: http://localhost:10220
+    interface: ":10220"
```

Comment:
No.
Now  users have to update port in multiple places.

Just use the svc port and update the config file automatically.